config.py

```diff
@@ -31,6 +31,7 @@ class Config(object):
     CASE_USERNAME = os.getenv('CASE_USERNAME')
     CASE_PASSWORD = os.getenv('CASE_PASSWORD')
     CASE_AUTH = (CASE_USERNAME, CASE_PASSWORD)
+    MAX_CASES_RETRIEVED_PER_SURVEY = os.getenv('MAX_CASES_RETRIEVED_PER_SURVEY', 12)
```

Comment:
I wouldn't set a default on this and instead would add it to our deployment config, but do whatever Chris did for the max survey retrieval thing he did

Reply:
You say that but he  literally changed 4 to 12 directly in the view, so I am not doing that . Why not set a default ? Its one less thing to worry about and we can change it at anytime by setting an environment variable. If we add it as an env then we have one more thing to maintain and we do not know if it will ever change ?

Reply:
I have a slight worry about this causing unintended behaviours down the line but am happy to let it go for now